### DISCUSSION TITLE
Translations for conference place and conference title (DOI import)

### DISCUSSION THREAD

**Discussion by vgerlach** [OPUS4/community#17] [Ideas]
So far the translations for the enrichments needed for DOI import need to be set manually. It would be helpful if some translations would be part of OPUS standard: e.g. conference_title, conference_place

Enrichmentconference_title
    de  Konferenztitel
    en  conference title
 
header_Enrichmentconference_title
    de  Konferenztitel 
    en  conference title
 
hint_Enrichmentconference_title
    de  Tragen Sie hier bitte den Konferenztitel ein   
    en  Please enter the conference title
 
button_label_add_one_moreEnrichmentconference_title
    de  Einen weiteren Konferenztitel hinzufügen   
    en  Add another conference title
 
button_label_deleteEnrichmentconference_title
    de  Den letzten Konferenztitel entfernen   
    en  Remove the last conference title
 
groupconference_title
    de  Konferenztitel 
    en  conference title
 
hint_groupconference_title
    de  Konferenztitel 
    en  conference title
 
Enrichmentconference_place
    de  Konferenzort
    en  conference place
 
header_Enrichmentconference_place
    de  Konferenzort
    en  conference place
 
hint_Enrichmentconference_place
    de  Bitte tragen Sie den Konferenzort ein
    en  Please enter the conference place
 
button_label_add_one_moreEnrichmentconference_place
    de  Einen weiteren Konferenzort hinzufügen
    en  Add another conference place
 
button_label_deleteEnrichmentconference_place
    de  Letzten Konferenzort entfernen
    en  Remove last conference place
 
groupconference_place
    de  Konferenzort
    en  conference place
 
hint_groupconference_place
    de  Konferenzort
    en  conference place

**Comment by alw-bsz** [comment]
Translations for the conference enrichments are not needed so far, as these fields have been removed from the DOI based metadata import.
